### PR TITLE
Add MXD tutorial 2 access policies

### PR DESCRIPTION
**Pull request by jkbquabeck**
## Description

This PR publishes the second MXD tutorial on access policies, formerly part of the third section in the current MXD doc. This was requested in issue #315 (https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/315). The MXD tutorials are to be published in different repositories, starting with tutorial_resources.

## Pre-review checks

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (paullatzelsperger) [inline mxd/3.2_MXD_tutorial2_Access_policies.md:5]: this table confuses rather than it explains, since the same information is contained in the JSON example further down. pls remove.

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (paullatzelsperger) [inline mxd/3.2_MXD_tutorial2_Access_policies.md:34]: please be more specific regarding "access" to the asset. we are defining an `accessPolicy`, which restricts the contents of the catalog. "Access" could also lead people to believe we mean "allowed to transfer"

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (paullatzelsperger) [inline mxd/3.2_MXD_tutorial2_Access_policies.md:72]:
```suggestion
Lastly, the asset and the access policy must be linked in a contract definition.
```

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (paullatzelsperger) [inline mxd/3.2_MXD_tutorial2_Access_policies.md:73]:
```suggestion
Action (Bob): Create a contract definition including the asset and the policies you´ve created. For this, use the following curl command:
```

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (paullatzelsperger) [inline mxd/3.2_MXD_tutorial2_Access_policies.md:83]: I would prefer to use different policies here. For example, it would be very illustrative, to have a _restrictive_ access policy, and a _permissive_ contract policy.

**Review by paullatzelsperger — CHANGES_REQUESTED** [review]

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (hemantxpatel) [reply mxd/3.2_MXD_tutorial2_Access_policies.md:77]: @jkbquabeck Please add a new section just above this one to create a contract policy. This is the curl:
```
curl --location 'http://localhost/bob/management/v2/policydefinitions' \
--header 'Content-Type: application/json' \
--header 'X-Api-Key: password' \
--data-raw '{
  "@context": {
    "odrl": "http://www.w3.org/ns/odrl/2/"
  },
  "@type": "PolicyDefinitionRequestDto",
  "@id": "5",
  "policy": {
    "@type": "Policy",
    "odrl:permission": [
      {
        "odrl:action": "USE",
        "odrl:constraint": {
          "@type": "LogicalConstraint",
          "odrl:or": [
            {
              "@type": "Constraint",
              "odrl:leftOperand": "BpnCredential",
              "odrl:operator": {
                "@id": "odrl:eq"
              },
              "odrl:rightOperand": "active"
            }
          ]
        }
      }
    ]
  }
}'
```

File: mxd/3.2_MXD_tutorial2_Access_policies.md
Comment (hemantxpatel) [inline mxd/3.2_MXD_tutorial2_Access_policies.md:78]: Update this as `"contractPolicyId": "5",`

**Review by hemantxpatel — APPROVED** [review]

**Comment by jkbquabeck** [comment]
@hemantxpatel Changed curl commands as requested. 
@stephanbcbauer once this PR is approved you can use the updated contents in the github.io repo feat/tutorials branch.
If approval is granted, do not merge this PR but close it, to avoid redundancy.

**Comment by hemantxpatel** [comment]
> @hemantxpatel Changed curl commands as requested.  

Thanks, it looks good now.

> @stephanbcbauer once this PR is approved you can use the updated contents in the github.io repo feat/tutorials branch. If approval is granted, do not merge this PR but close it, to avoid redundancy.

I am little out of context here. Why don't we want to merge this PR. It looks good to me.

**Review by paullatzelsperger — APPROVED** [review]

**Pull request merged**
